src/librustdoc/json/mod.rs

```diff
@@ -81,24 +80,26 @@ impl JsonRenderer<'_> {
             .unwrap_or_default()
     }
 
-    fn get_trait_items(&mut self, cache: &Cache) -> Vec<(types::Id, types::Item)> {
-        cache
+    fn get_trait_items(&mut self) -> Vec<(types::Id, types::Item)> {
+        self.cache
+            .clone()
```

Comment:
Why is this cloned?

Reply:
Because `cache` is used in a closure afterwards. The cost of clning `cache` is low though since it's a `Rc`.

Reply:
Ok, can you make that clear by using `Rc::clone`?

Reply:
Sure!